## 🎬 **Análise por Variável Determinante**

---

### 🔮 **1. Efeitos Visuais + CGI Alto Nível (VFX)**
---



**Impacto no Custo:**
- Um dos elementos mais caros da produção, especialmente em filmes de ficção científica ou fantasia.
- Efeitos visuais de alta qualidade requerem tecnologia avançada e profissionais especializados, aumentando significativamente o orçamento.

**Impacto no Lucro:**
- VFX impressionantes podem atrair grandes audiências, especialmente em filmes de ação e ficção científica, potencializando a bilheteria.

**Impacto no Desenvolvimento:**
- A produção de VFX demanda tempo e coordenação entre diversas equipes, podendo prolongar o cronograma de produção.

**Exemplos:**
- **❌ VFX Mal Avaliado:** *Cats* (2019) - Efeitos visuais perturbadores com design "uncanny valley"
  - 🔗 [The Guardian](https://www.theguardian.com/film/2019/dec/22/cats-film-a-disaster-of-hair-raising-proportions)

- **✅ VFX Bem Avaliado:** *Avatar* (2009) - Revolução no CGI 3D
  - 🔗 [BBC News](https://www.bbc.com/news/entertainment-arts-50812799)

- **✅ VFX Bem Avaliado:** *Duna 2* (2023) - Oscar de Melhores Efeitos Visuais
  - 🔗 [Oscars.org](https://www.oscars.org/oscars/ceremonies/2024)

### 🔊 **2. Design de Som + Mixagem Imersiva (Dolby Atmos / IMAX Sound)**
---

**Impacto no Custo:**
- Requer mixagem profissional e equipamentos de ponta, encarecendo a pós-produção.
- A implementação de som imersivo exige equipamentos especializados e profissionais experientes, elevando os custos de pós-produção.

**Impacto no Lucro:**
- Um design de som de alta qualidade pode melhorar a experiência do espectador, aumentando a satisfação e o boca a boca positivo.

**Impacto no Desenvolvimento:**
- Requer planejamento detalhado desde a gravação até a mixagem final, podendo estender o tempo de pós-produção.

**Exemplos:**
- **❌ Som Mal Avaliado:** *Transformers: Age of Extinction* (2014) - Mixagem caótica
  - 🔗 [Rotten Tomatoes](https://www.rottentomatoes.com/m/transformers_age_of_extinction)

- **✅ Som Bem Avaliado:** *Dunkirk* (2017) - Oscar de Melhor Mixagem de Som
  - 🔗 [Oscars.org](https://www.oscars.org/oscars/ceremonies/2018)

### 🎨 **3. Fotografia + Direção de Arte**
---

**Impacto no Custo:**
- Inclui cenografia, figurino e iluminação — com grande peso orçamentário.
- Investimentos em cenografia, figurinos e design visual aumentam os custos, mas são essenciais para criar uma estética envolvente.

**Impacto no Lucro:**
- Uma direção de arte distinta pode tornar o filme memorável, influenciando positivamente a crítica e o público.

**Impacto no Desenvolvimento:**
- Exige colaboração entre departamentos e planejamento meticuloso para garantir coesão visual.

**Exemplos:**
- **❌ Direção de Arte Mal Avaliada:** *Suicide Squad* (2016) - Estética confusa
  - 🔗 [Variety](https://variety.com/2016/film/reviews/suicide-squad-review-margot-robbie-1201831546/)

- **✅ Direção de Arte Bem Avaliada:** *The Grand Budapest Hotel* (2014) - Oscar de Direção de Arte
  - 🔗 [IndieWire](https://www.indiewire.com/gallery/best-cinematography-movies-shots/)

### ✂️ **4. Edição Rítmica + Montagem**
---

**Impacto no Custo:**
- Embora menos onerosa que outras variáveis, uma edição eficaz requer editores experientes e tempo dedicado, influenciando o orçamento.

**Impacto no Lucro:**
- Uma montagem bem executada mantém o ritmo e o engajamento do público, impactando positivamente a recepção do filme.

**Impacto no Desenvolvimento:**
- Crucial na pós-produção para manter ritmo e coesão narrativa.
- A edição é essencial para moldar a narrativa e corrigir falhas de gravação.

**Exemplos:**
- **❌ Edição Mal Avaliada:** *The Snowman* (2017) - Cenas faltando e edição incoerente
  - 🔗 [The Verge](https://www.theverge.com/2017/10/13/16471250/the-snowman-movie-bad-editing-missing-scenes)

- **✅ Edição Bem Avaliada:** *Mad Max: Fury Road* (2015) - Oscar de Melhor Edição
  - 🔗 [The Hollywood Reporter](https://www.hollywoodreporter.com/movies/movie-news/mad-max-fury-road-editing-oscar-864072/)

### 🕶️ **5. Tecnologia 3D/4DX (IMAX / 4K / HFR)**
---

**Impacto no Custo:**
- Pode justificar ingressos mais caros e atrair público para experiências cinematográficas imersivas.
- A adoção de tecnologias avançadas implica em equipamentos especializados e processos de produção mais complexos, elevando os custos.

**Impacto no Lucro:**
- Quando bem utilizadas, essas tecnologias podem oferecer experiências imersivas, atraindo mais espectadores e justificando preços de ingressos mais altos.

**Impacto no Desenvolvimento:**
- Exige planejamento desde a pré-produção para garantir que as tecnologias sejam integradas de forma eficaz à narrativa.

**Exemplos:**
- **❌ Tecnologia Mal Avaliada:** *The Hobbit: An Unexpected Journey* (2012) - HFR (48fps) artificial
  - 🔗 [Wired](https://www.wired.com/2012/12/hobbit-hfr-48fps-controversy/)

- **✅ Tecnologia Bem Avaliada:** *Interstellar* (2014) - Uso exemplar de IMAX
  - 🔗 [IMDb](https://www.imdb.com/title/tt0816692/technical)

## 📊 **Manipulando Tabelas**
---

In [3]:
import pandas as pd

fatores_lucro_sp = "spreadsheets/fatores_lucro.xlsx"
df_vd = pd.read_excel(fatores_lucro_sp, sheet_name="variaveis_determinantes")
df_cv = pd.read_excel(fatores_lucro_sp, sheet_name="combinacoes_variaveis")

display(df_vd)

Variável Determinante                     Filme Exemplo  \
0       Efeitos Visuais (VFX)  Avatar: O Caminho da Água (2022)   
1   Som Imersivo (IMAX/Dolby)                       Duna (2021)   
2  Direção de Arte/Fotografia                 O Rei Leão (2019)   
3              Edição Rítmica       Velozes & Furiosos 7 (2015)   
4         3D / 60K Tecnologia                  Gravidade (2013)   

  Custo Min (US$) Custo Médio (US$) Custo Máx (US$) Receita Min (US$)  \
0            350M              400M            460M              2.0B   
1            150M              165M            165M              400M   
2            250M              260M            260M              1.6B   
3            190M              190M            190M              1.5B   
4            100M              110M            110M              723M   

  Receita Média (US$) Receita Máx (US$) Lucro Médio (US$)  
0               2.32B             2.32B             1.92B  
1              410.7M            410.7M            245.7M  
2               1.66B             1.66B              1.4B  
3               1.52B             1.52B             1.33B  
4              723.2M            723.2M            613.2M

In [4]:
display(df_cv)

Combinação de Variáveis                     Filme Exemplo  \
0                   VFX + IMAX + 3D  Avatar: O Caminho da Água (2022)   
1  Direção de Arte + Edição Rítmica       Velozes & Furiosos 7 (2015)   
2            3D + Duração Otimizada                  Gravidade (2013)   
3             VFX + Direção de Arte                 O Rei Leão (2019)   
4      IMAX + Som Imersivo + Edição                       Duna (2021)   

  Custo Min (US$) Custo Médio (US$) Custo Máx (US$) Receita Min (US$)  \
0            350M              400M            460M              2.0B   
1            190M              190M            190M              1.5B   
2            100M              110M            110M              723M   
3            250M              260M            260M              1.6B   
4            150M              165M            165M              400M   

  Receita Média (US$) Receita Máx (US$) Lucro Médio (US$)  
0               2.32B             2.32B             1.92B  
1               1.52B             1.52B             1.33B  
2              723.2M            723.2M            613.2M  
3               1.66B             1.66B              1.4B  
4              410.7M            410.7M            245.7M

### 📚 Fontes Utilizadas
---

#### 🎬 **Avatar: O Caminho da Água (2022)**
- **Orçamento:** US$ 400 milhões  
- **Bilheteria Mundial:** US$ 2,320,250,281  
- **Fontes:**  
  - [The Numbers](https://www.the-numbers.com/movie/Avatar-The-Way-of-Water-%282022%29)  
  - [Box Office Mojo](https://www.boxofficemojo.com/title/tt1630029/)

---

#### 🎬 **Duna (2021)**
- **Orçamento:** US$ 165 milhões  
- **Bilheteria Mundial:** US$ 410,668,018  
- **Fontes:**  
  - [The Numbers](https://www.the-numbers.com/movie/Dune-%282020%29)  
  - [Box Office Mojo](https://www.boxofficemojo.com/title/tt1160419/)

---

#### 🎬 **O Rei Leão (2019)**
- **Orçamento:** US$ 260 milhões  
- **Bilheteria Mundial:** US$ 1,662,899,439  
- **Fontes:**  
  - [The Numbers](https://www.the-numbers.com/movie/Lion-King-The-%28Live-Action%29-%282019%29)  
  - [Box Office Mojo](https://www.boxofficemojo.com/release/rl3321923073/)

---

#### 🎬 **Velozes & Furiosos 7 (2015)**
- **Orçamento:** US$ 190 milhões  
- **Bilheteria Mundial:** US$ 1,515,341,399  
- **Fontes:**  
  - [The Numbers](https://www.the-numbers.com/movie/Furious-7)  
  - [Box Office Mojo](https://www.boxofficemojo.com/showdown/sd1816000004/)

---

#### 🎬 **Gravidade (2013)**
- **Orçamento:** US$ 110 milhões  
- **Bilheteria Mundial:** US$ 723,192,705  
- **Fontes:**  
  - [The Numbers](https://www.the-numbers.com/movie/Gravity-%282013%29)  
  - [Box Office Mojo](https://www.boxofficemojo.com/release/rl89622017/)
